modules/ml/src/main/java/org/apache/ignite/ml/environment/DefaultLearningEnvironmentBuilder.java

```diff
@@ -34,6 +34,9 @@
     /** Serial version id. */
     private static final long serialVersionUID = 8502532880517447662L;
 
+    /** Default partition data TTL (infinite). */
+    private static final long INFINITE_TTL = -1;
```

Comment:
ENUM - ENUM, it sounds like ENUM, you love old C style)))

Reply:
No, it doesn't look like a `enum`.